record_tx.go

```diff
@@ -88,10 +88,10 @@ func waitForRecordTxWriteLock(ctx context.Context, c ClientInterface, key string
 	// Relevant for bux to bux transactions, as we have 1 tx but need to record 2 txs - outgoing and incoming
 
 	lockKey := fmt.Sprintf(lockKeyRecordTx, key)
-	c.Logger().Info(ctx, lockKey)
+	c.Logger().Info().Msgf(lockKey)
 
 	// TODO: change to DEBUG level log when we will support it
-	c.Logger().Info(ctx, fmt.Sprintf("try add write lock %s", lockKey))
+	c.Logger().Info().Msgf("try add write lock %s", lockKey)
```

Comment:
Is it possible to use `zerolog.Debug` here ?

Reply:
Done